src/handlers/relabel.rs

```diff
@@ -76,8 +76,18 @@ async fn handle_input(
     let mut changed = false;
     for delta in &input.0 {
         let name = delta.label().as_str();
-        if let Err(msg) = check_filter(name, config, &event.user(), &ctx.github).await {
-            let cmnt = ErrorComment::new(&event.issue().unwrap(), msg.to_string());
+        let err = match check_filter(name, config, is_member(&event.user(), &ctx.github).await) {
+            Ok(CheckFilterResult::Allow) => None,
+            Ok(CheckFilterResult::Deny) => Some(format!("Label {} can only be set by Rust team members", name)),
+            Ok(CheckFilterResult::DenyUnknown) => Some(format!(
+                "Label {} can only be set by Rust team members;\
+                 we were unable to check if you are a team member.",
+                 name
+            )),
+            Err(err) => Some(err.to_string()),
```

Comment:
We probably want to have check_filter return `Result<CheckFilterResult, String>` or something like that instead here to avoid the potentially not great conversion to string here (since we send this to the user in a GH comment)

Reply:
Is 818e6b4 what you had in mind?